jmh-benchmarks/src/main/java/org/apache/kafka/jmh/streams/MockMemoryLRUCache.java

```diff
@@ -0,0 +1,275 @@
+/**
```

Comment:
Is there a reason we need this instead of `org.apache.kafka.common.cache.LRUCache`?